global.json

```diff
@@ -1,18 +1,18 @@
 {
-  "sdk": {
-    "version": "10.0.100-alpha.1.24573.1",
-    "allowPrerelease": true,
-    "rollForward": "major"
-  },
-  "tools": {
-    "dotnet": "10.0.100-alpha.1.24573.1",
-    "runtimes": {
-      "dotnet": [
-        "$(MicrosoftNETCoreAppVersion)"
-      ]
+    "sdk": {
+      "version": "10.0.100-alpha.1.24573.1",
+      "allowPrerelease": true,
+      "rollForward": "major"
+    },
+    "tools": {
+      "dotnet": "10.0.100-alpha.1.24573.1",
+      "runtimes": {
+        "dotnet": [
+          "$(MicrosoftNETCoreAppVersion)"
+        ]
+      }
+    },
+    "msbuild-sdks": {
+      "Microsoft.DotNet.Arcade.Sdk": "10.0.0-beta.24606.6"
     }
```

Comment:
Is this change intentional? It looks like indentation change only (and wrong indentation it seems)

Reply:
No, I messed up ident.

Reply:
It is still showing up in the diff